### DISCUSSION TITLE
"The workflow is not valid. Object reference not set to an instance of an object."

### DISCUSSION THREAD

**Discussion by 2called-chaos** [community/community#27719] [Actions] [answered]
https://github.com/2called-chaos/fuzzily/actions/runs/2755155360/workflow

Can somebody tell me what that means? It appears to be a git error message. I can run the thing with act locally so I assume the yaml itself should be somewhat valid.

I suspect it being a fork is maybe the problem? Is there something I can do?

Thanks in advance.

**Comment by airtower-luna** [comment, marked as answer]
Your workflow seems to have an entry in `jobs.build.services` that doesn't actually define a service: the `sqlite3` one. You [need to define an `image` to run](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservices), otherwise there can't be a service. And I know SQLite isn't a service you can run, it's a library that uses a local file as DB storage.

I'm not familiar with your tools so I can't say what you should do instead to test with SQLite (probably something in a step), but removing the non-service should get you one step further. :smiley_cat:

**Reply by 2called-chaos** [reply]
Thank you very much, I tried to abuse it for the env part which wouldn't have worked anyway :)
